### PR TITLE
Add OpenTracing TCK FAT

### PR DESCRIPTION
**Pull request by kgibm**

### CODE REVIEW

**Comment by kgibm** [comment]
#build

File: dev/com.ibm.ws.opentracing_fat/publish/tckRunner/tck/blank-tck-suite.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!--Copyright (c) 2017 IBM Corporation and others. All rights reserved.
```
Comment (hutchig) [inline dev/com.ibm.ws.opentracing_fat/publish/tckRunner/tck/blank-tck-suite.xml]: You don't need one of these file. I just had one as an example as my main tck-suite had a
beanshell exclude in it.

File: dev/com.ibm.ws.opentracing_fat/publish/servers/OpentracingTCKServer/server.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+<server>
+    <featureManager>
+        <feature>servlet-3.1</feature>
+        <feature>componenttest-1.0</feature>
+        <feature>webProfile-7.0</feature>
+        <feature>localConnector-1.0</feature>
+        <feature>cdi-1.2</feature>
+	<feature>opentracing-1.0</feature>
+	<feature>mpOpenTracing-1.0</feature>
+	<feature>usr:opentracingMock-0.30</feature>
```
Comment (hutchig) [inline dev/com.ibm.ws.opentracing_fat/publish/servers/OpentracingTCKServer/server.xml]: I do not know enough to check that this unusual <feature>usr:opentracingMock-0.30</feature>
is not a locla unit testing feature that only lives on a developers laptop? Or is fine. :-)

Comment (kgibm) [reply dev/com.ibm.ws.opentracing_fat/publish/servers/OpentracingTCKServer/server.xml]: Thanks Gordon for the review! I removed blank-tck-suite.xml. The user feature is expected as it has the Tracer under test, and it's built by the FAT itself (personal build passed). Thanks.

**Pull request merged**